aixplain/modules/model/__init__.py

```diff
@@ -84,6 +85,7 @@ def __init__(
         self.backend_url = config.BACKEND_URL
         self.function = function
         self.is_subscribed = is_subscribed
+        self.createdAt = createdAt
 
     def to_dict(self) -> Dict:
         """Get the model info as a Dictionary
```

Comment:
Please check whether this `to_dict` method works with a datetime attribute.

Reply:
It seems to work, this is what it returns:
{'id': '64e615671567f848804985e1', 'name': 'GPT2', 'supplier': <Supplier.OPENAI: {'id': 1777, 'name': 'OpenAI', 'code': 'openai'}>, 'additional_info': {'parameters': {'language': ['en']}}, 'createdAt': datetime.datetime(2023, 8, 30, 10, 53, 10, 827000, tzinfo=datetime.timezone.utc)}
I added createdAt to to_dict